### DISCUSSION TITLE
IDP Microsoft Edge & Chrome printing login form

### DISCUSSION THREAD

**Discussion by zelytra** [keycloak/keycloak#28298] [Q&A] [answered]
I have a problem with Microsoft IDP implementation, when i use Firefox everythings work ! The user is redirect to Keycloak then to the Microsoft login page, and then auth in my app. 

But when i use **Edge** or **Chrome**, a stupid login pop up appear to ask my user to log in directly in the browser, you can close it and then it goes on, it didn't interfer with the login process... How to prevent to this pop up to open or trigger ?

Edge : 
![image](https://github.com/keycloak/keycloak/assets/72125876/9a8b8e2e-128d-4d17-9194-3bc19cc967e5)

Chrome:
![image](https://github.com/keycloak/keycloak/assets/72125876/660ac897-e1fc-4a07-876d-da302a6d871b)

**Comment by codespearhead** [comment]
This is a basic login prompt. Try [this](https://answers.microsoft.com/en-us/microsoftedge/forum/all/microsoft-edge-shows-windows-security-popup-to-log/871a94cd-a3f7-4f73-9aa3-dacaa2ca7749).

**Reply by zelytra** [reply, marked as answer]
I dont want to disable it on my side, but for all user. This prompt appear by default. But is okay, i completly reset my keycloak config and reconfigure it and now i dont have it anymore

**Reply by codespearhead** [reply]
Can you [close this discussion](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) then?
